testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,83 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}', '$_SAMPLE']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}-${COMMIT_SHA}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}'
+  - '--cluster'
+  - '${_CLUSTER}'
+  - '--cluster-location'
+  - '${_SERVICE_REGION}'
+  - '--platform'
+  - 'gke'
+
+- id: 'Get Cloud Run URL'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_url() {
+        gcloud run services describe ${_SERVICE}-${COMMIT_SHA} --format 'value(status.url)' \
+          --platform gke --cluster ${_CLUSTER} --cluster-location ${_SERVICE_REGION}
+    }
+    echo $(get_url) > _service_url
+    echo "Cloud Run URL for ${_SERVICE} is $(cat _service_url)"
+
+- id: 'Get Cloud Run External IP'
+  name: 'gcr.io/cloud-builders/kubectl'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_ip() {
+        gcloud container clusters get-credentials ${_CLUSTER} --zone ${_SERVICE_REGION} --project ${PROJECT_ID} \
+        && kubectl get svc istio-ingress --namespace gke-system -o yaml | sed -n -e 's/    - ip: //p'
+    }
+    echo $(get_ip) > _service_ip
+
+- id: 'Integration Tests'
+  # TODO: Update the following image name, entrypoint, and args to fit your testing needs
+  name: 'alpine:latest'
+  entrypoint: '/bin/sh'
+  dir: $_SAMPLE
+  args:
+  - '-c'
+  - |
+    echo "Add integration tests!"
+
+- id: 'Teardown'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    set -x
+    gcloud --quiet container images delete gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}
+    gcloud --quiet run services delete ${_SERVICE}-${COMMIT_SHA} --platform gke
+    set +x
+    echo "View build details in the console: https://console.cloud.google.com/cloud-build/builds/${BUILD_ID}"
+
+# Uncomment if skipping teardown to associate build with container image.
+# images:
+# - 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}'
+
+# TODO: Update these User-defined substitutions
+substitutions:
+  _SAMPLE: SAMPLE_FOLDER_NAME
+  _SERVICE: SERVICE_NAME
+  _CLUSTER: CLUSTER_NAME
+  _CLOUDSDK_VERSION: latest
+  _SERVICE_REGION: us-central1-a
```

Comment:
I like the consistency between templates, but maybe aligning with gcloud would be more intuitive for CRfA developers? `CLUSTER_LOCATION` in that case.